app/controllers/application_controller.rb

```diff
@@ -1,10 +1,28 @@
 require './config/environment'
+require "./app/models/user"
+require 'pry'
 
 class ApplicationController < Sinatra::Base
 
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "password_security"
   end
 
-end
\ No newline at end of file
+    get '/' do
+      erb :'index'
+    end
+
+    	helpers do
+    		def logged_in?
+    			!!session[:user_id]
```

Comment:
You can use your `logged_in?` helper method here, which, when it calls a memoized value of current_user, is preferable to referring to the session object directly.